pkgs/development/ocaml-modules/logs/default.nix

```diff
@@ -16,21 +16,17 @@
   lwtSupport ? true,
   cmdlinerSupport ? true,
 }:
-let
-  pname = "logs";
-  webpage = "https://erratique.ch/software/${pname}";
-in
```

Comment:
Also here, inlining these definitions doesn't improve anything.

Reply:
The latest idea on the topic was to make things not depend on `finalAttrs` (https://github.com/NixOS/nixpkgs/pull/387725#issuecomment-2924073216) since the user *may* want to change it and it *may* be confusing what repercussions this may have. This `let` isn't affected by this reasoning as it is completely static, i.e. nothing changes by inlining it, except new duplication. It is also a very common pattern in `ocamlPackages`.

If you can show that changing this has a performance impact beyond the margin of error, I'm going to be impressed.

Reply:
(I'm not a maintainer of any of this code. Just a drive by commenter)

> It is also a very common pattern in `ocamlPackages`.

I personally find this to be the best guideline. If there isn't a clear rule/guideline, then try to be consistent with the nearby code. Let's instead spend humanity's time on building linters.

That said, I'm having trouble convincing myself that `let pname = ...` is a common pattern in `pkgs/development/ocaml-modules/` (not gonna share the unsound grepping I did to try to test this). Perhaps I'm misunderstanding what we're talking about here.